src/main/java/seedu/address/logic/parser/mode/ModeParser.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.logic.parser.mode;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.commons.core.Messages.MESSAGE_UNKNOWN_COMMAND;
+import static seedu.address.logic.parser.person.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.person.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.person.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.person.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.person.CliSyntax.PREFIX_TAG;
+
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+import seedu.address.logic.commands.mode.Command;
+import seedu.address.logic.commands.mode.SwitchCommand;
+import seedu.address.logic.commands.person.AddCommand;
+import seedu.address.logic.commands.person.HelpCommand; // take this out of Person
```

Comment:
Maybe we can add a TODO keyword in the comment so it's easier for us to locate in the future
i.e. `// TODO take this out of Person `